serverlessworkflow/modules/ROOT/pages/cloud/build-workflow-image-with-quarkus-cli.adoc

```diff
@@ -216,7 +225,7 @@ You can notice that, the startup of native image is faster than the JVM image.
 .Example request
```

Comment:
I think request and response can be changed into tabs? what do you think?

Reply:
sure